csrc/index_compute.cpp

```diff
@@ -1625,17 +1625,18 @@ std::vector<Val*> Index::getNonGlobalProducerStridedIndices(
       continue;
     }
 
+    auto override_it = override_index.find(root_dom[i]);
```

Comment:
This is a bug fix. It was fine just because the non-global wasn't happening before. The override check should be done before the assertion at line 1631.